corpus/segments.py

```diff
@@ -261,6 +262,33 @@ def run(self):
                 f.writelines(lines[start:end])
 
 
+class DynamicSplitSegmentFileJob(Job):
+
+    def __init__(self, segment_file, concurrent=1):
+        self.segment_file = segment_file
+        self.concurrent = concurrent
+        self.segment_dir = self.output_path("split", directory=True)
```

Comment:
output members should have prefix out_ in the name

Reply:
I hope we can write those constrains/requirements (also that regarding documentation) as a guideline. This will save a lot of time in the future.

Reply:
We have this here: https://github.com/rwth-i6/i6_core/blob/main/CONTRIBUTUNG.md,
but of course this can be extended if something is unclear.

Reply:
sorry for missing the info you already mentioned above